protokube/pkg/protokube/volume_mounter.go

```diff
@@ -52,6 +52,12 @@ func (k *VolumeMountController) mountMasterVolumes() ([]*Volume, error) {
 	}
 
 	for _, v := range attached {
+		// Do not ever try to mount root devices
+		if v.LocalDevice == "/dev/sda1" || v.LocalDevice == "/dev/xvda" {
```

Comment:
Yes it should not be needed, but our code will go into an infinite loop if you try to mount /dev/sda1 which is the drive in the rhel launchconfig. There are more details in the issue, but we should not ever try to mount either of these partitions, and I would like code to safeguard that from occuring.

I can pull it, but 'shouldn't' has got me in trouble a bunch,